### ISSUE TITLE
Consistency with quoting or not quoting yaml strings in docs

### ISSUE THREAD

**Issue by dingobar** [linkedin/datahub#2821]
**Describe the bug**
In the docs, [particularly the sources/sinks section of the ingestion page](https://datahubproject.io/docs/metadata-ingestion/), alternates between quoting and not quoting strings in `yaml`. I've played around with it a bit now and it seems like there is not reason to quote strings (I assume because `pyyaml` ignores the quotes). Therefore, I suggest either quoting all strings in the docs or removing all of them (I prefer the latter) in order not to confuse new users.

```yml
sink:
        type: datahub-kafka
        config:
                connection:
                        bootstrap: localhost:9092
                        schema_registry_url: http://localhost:8081
```

is the same as 

```yml
sink:
        type: "datahub-kafka"
        config:
                connection:
                        bootstrap: "localhost:9092"
                        schema_registry_url: "http://localhost:8081"
```

In other words, this can be confusing
![image](https://user-images.githubusercontent.com/41419288/124269858-b9b2fa80-db3b-11eb-934c-c412359777e5.png)

**Comment by hsheth2** [comment]
All correctly implemented yaml parsers will ignore the quotes in most cases, since it's part of the YAML spec. However, there are some cases where there's a minor risk of the string being misinterpreted as a different type (e.g. `yes` will become a bool unless quoted). 

It likely doesn't make sense to change all the documentation for aesthetic choice, but if you feel strongly about it, please feel free to open a PR!

**Comment by dingobar** [comment]
Hey, sorry for the issue spam, I was working through the docs and thought I'd raise an issue whenever I found any. But I'll gather them up for later instead when I have some more time to contribute 👍
